Makefile

```diff
@@ -41,79 +45,39 @@ fix: ## Fixup files in the repo.
 
 .PHONY: clean
 clean: ## Cleanup build artifacts and tool binaries.
-	rm -rf $(BUILD_DIR) dist tools/bin
```

Comment:
should we not ignore the tools/ dir ?

Reply:
`tools/bin` is already ignored. We can't ignore the whole `tools/` directory since it contains scripts.